libraries/helpers.rb

```diff
@@ -45,6 +45,17 @@ def install_mixlib_versioning
       require 'mixlib/versioning'
     end
 
+    def install_mixlib_install
+      # TODO: Change this code to install mixlib-install from rubygems before merge.
+      chef_gem "#{new_resource.product_name}-mixlib-install" do # ~FC009 foodcritic needs an update
+        package_name 'mixlib-install'
+        compile_time true
+        source '/home/vagrant/mixlib-install/pkg/mixlib-install-0.8.0.gem'
+      end
```

Comment:
Might be worth replicating the pattern we use in the artifactory-pro cookbook:
https://github.com/chef-cookbooks/artifactory-pro/blob/master/libraries/_helper.rb#L199-L285